TheForceEngine/TFE_DarkForces/GameUI/pda.cpp

```diff
@@ -1,5 +1,5 @@
 #include <cstring>
-
+#include <SDL.h>
```

Comment:
You should no longer need this include.

Reply:
SDL is now removed in [bda14bc](https://github.com/luciusDXL/TheForceEngine/pull/459/commits/bda14bc28b486a53c034c4c8f425474dc863a6d1)